src/components/DevelopmentServerPane/DevelopmentServerPane.js

```diff
@@ -92,6 +104,20 @@ class DevelopmentServerPane extends PureComponent<Props> {
               {docLink}
             </InfoWrapper>
             <TerminalWrapper>
+              <ButtonWrapper>
+                <Title>Server Logs</Title>
```

Comment:
We have a `Heading` component that I think might be useful here. It has a `size` prop, but the smallest size is 26px, which might be too big.

If you could swap it out that'd be awesome :+1: after merging I'll take a closer look at the design and maybe add a new `xsmall` size to make it 21px or 22px, if necessary.